StorySquad/SubmissionsAPI.swift

```diff
@@ -0,0 +1,429 @@
+//
+//  SubmissionsAPI.swift
+//  StorySquad
+//
+//  Created by Bohdan Tkachenko on 6/7/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// Put in global file
+enum HeaderNames: String {
+    case authorization = "Authorization"
+    case contentType = "Content-Type"
+}
+
+// Put in global file
+enum NetworkingError: Error {
+    case noBearer
+    case serverError(Error)
+    case badDecode
+    case unauthorized
+    case noData
+    case unexpectedStatusCode(Int)
+    case badEncoding
+    case noRepresentation
+    case formattedJSONIncorrectly
+    case missingRequiredElement
+}
+
+class SubmissionsAPI {
+    
+    private let baseURL = URL(string: "https://storysquad-api.herokuapp.com/")!
+    
+    var parentUser: Parent?
+    var children: [Child]?
+    var childUser: Child?
+    
```

Comment:
Add a property to store the current submission (var submission: Submission?) and instantiate it in getChildSubmissionFromDataBase. Then you can access the submission id from that property instead of passing it into each function, and also update it as the child goes through the submission process. Also this property will need to be available to display the reading part in the UI.